src/parser/druid/restoration/constants.js

```diff
@@ -15,19 +15,19 @@ export const ABILITIES_AFFECTED_BY_HEALING_INCREASES = [
   SPELLS.RENEWAL_TALENT.id,
   SPELLS.SPRING_BLOSSOMS.id,
   // The following spells don't double dip in healing increases.
-  SPELLS.LEECH.id,
+  SPELLS.LEECH,
```

Comment:
This mismatches the lines above

Reply:
I coulda sworn I undid those changes before I commited, let me fix that real quick